server/routerlicious/packages/services/src/mongodb.ts

```diff
@@ -13,6 +13,8 @@ import { MongoErrorRetryAnalyzer } from "./mongoExceptionRetryRules";
 const MaxFetchSize = 2000;
 const MaxRetryAttempts = 3;
 const InitialRetryIntervalInMs = 1000;
+const errorSanitizationMessage = "REDACTED";
+const errorJsonKeyToSanitize = "data";
```

Comment:
apart from data, shouldn't we also redact data under any key with name ="contents"?

Reply:
so far, I did not find any samples where the data under key with name "contents" is customer content, but we can certainly redact that as well to be safe.